web-client.html

```diff
@@ -39,25 +39,39 @@
 
 </li>
 
-<li><p class="first">To view an image in the
-    <span class="bold">Image Viewer</span>, select the image in the data tree
-    and click on the <span class="bold">Launch Full Viewer</span> button.</p>
-
-<p>Alternatively, double click on the image or thumbnail, or right-click and
-    use the <span class="bold">Open With...</span> menu to select the
-    OMERO.web app to use (this feature is new in OMERO 5.3.0 and any
-    additional apps such as OMERO.figure will show up here once your system
-    administrator has installed and configured them to do so).</p>
-
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
-
-<img alt="../images/webclientView.jpg" class="align-center" src="images/webclientView.jpg" style="width: 100%;" />
-
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
+<div class="anchor" id="open_with"></div>
+
+<li>
+    <p class="first">
+        To view an image in the <span class="bold">Image Viewer</span>, select
+        the image in the data tree and click on the <span class="bold">Launch
+        Full Viewer</span> button.
+    </p>
+
+    <p>
+        Alternatively, double click on the image or thumbnail, or right-click
```

Comment:
maybe hyphenate double-click too if we do right-click (not absolutely sure which is best tbh)